vuepress/docs/next/tutorials/create/ms/run-local.md

```diff
@@ -2,11 +2,12 @@
 sidebarDepth: 2
 ---
 # Run Blueprint-generated Microservices and Micro Frontends in Dev Mode
-## Overview
-This tutorial guides you through running an Entando project with microservice and micro frontends in a local development environment. If you haven't generated an Entando project yet, start with the [Generate Microservices and Micro Frontend](./generate-microservices-and-micro-frontends.md) tutorial.
+This tutorial guides you through running an Entando project with microservices and micro frontends in a local development environment. If you haven't generated an Entando project yet, start with the [Generate Microservices and Micro Frontends](./generate-microservices-and-micro-frontends.md) tutorial.
 
 The steps below assume you are working in the top-level project directory.
 
+This tutorial is specific to bundles generated with the Entando JHipster Blueprint, which is a Git-based or a v1 bundle. If you have Docker-based project (v5), created on Entando 7.1 or newer, please refer to the release document for details. 
```

Comment:
@nshaw, I'll work on the comparison document, whether it's a table in the release or a separate doc, and link it in later. I feel like the information needs to be processed a bit with a few tutorials to figure out how best to present it.